src/picongpu/include/initialization/InitialiserController.hpp

```diff
@@ -87,6 +87,25 @@ class InitialiserController : public IInitPlugin
         log<picLog::SIMULATION_STATE > ("Loading from persistent data finished");
     }
 
+    /** log omega_p for each species
+     *
+     * calculate omega_p for each given species and create a `picLog::PHYSICS`
+     * log message
+     */
+    template<typename T_Species = bmpl::_1>
+    struct LogOmegaPe
```

Comment:
`LogOmegaP` right? Without `e` because that indicates specifically electrons, correct?